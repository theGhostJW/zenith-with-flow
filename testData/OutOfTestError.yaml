elementType: OutOfTestErrors
issues:
  - name: out of test
    infoType: OutOfTest
    warnings:
      - timestamp: '2017-11-25 11:28:22'
        level: warn
        subType: Message
        message: warning added for testing
    errors:
      - timestamp: '2017-11-25 11:28:22'
        level: error
        subType: Message
        popControl: NoAction
        message: bodged out of test example failure
        additionalInfo: blahh
    type2Errors: []
    knownDefects: []
