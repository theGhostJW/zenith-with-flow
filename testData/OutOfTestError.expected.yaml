################################################################################
#%######################## Out of Test Issues ##################################
################################################################################

warnings:
  - timestamp: 2017-11-25 11:28:22
    message: warning added for testing

errors:
  - timestamp: 2017-11-25 11:28:22
    message: bodged out of test example failure
    additionalInfo: blahh
